lib/learn_test/strategies/none.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module LearnTest
+  module Strategies
+    class None < LearnTest::Strategy
+      def service_endpoint
+        '/e/flatiron_none'
+      end
+
+      def detect
+        runner.files.include?('.canvas')
+      end
```

Comment:
This check will ensure that this new strategy applies only to code-alongs that originated in Canvas, while those that came from Learn will continue to behave as they have (i.e. no-op with a message that there are no specs).